<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PlayWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-30</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>4</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(184, 217, 138);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>85</y>
      <width>341</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(5, 119, 161);
border-radius:40px;
border-style:solid;
border-width:3px;
border-color: rgb(184, 88, 44);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="language_label">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>130</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(205, 133, 63);</string>
    </property>
    <property name="text">
     <string>LANGUAGE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="word_label">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>231</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(205, 133, 63);</string>
    </property>
    <property name="text">
     <string>WORDS</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>40</y>
      <width>41</width>
      <height>381</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>7</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QProgressBar{
	background-color: rgb(255, 255, 255);
	color: rgb(170, 0, 0);
border-radius:15px;
text-align:center;
}
QProgressBar::chunk{
border-radius:15px;
	background-color: rgb(205, 133, 63);
margin:1px;
height:20px;
}</string>
    </property>
    <property name="maximum">
     <number>20</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="timelabel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(135, 185, 225);
border-radius:50px;
border-style:solid;
border-width:3px;
color: rgb(170, 0, 0);
border-color: rgb(207, 104, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(135, 185, 225);
	color: rgb(170, 0, 0);
}
QPushButton::hover{
	background-color: rgb(5, 119, 161);
	color: rgb(255, 255, 255);
}
QPushButton::pressed{
	background-color: rgb(36, 83, 109);
    padding-left:5px;
	color: rgb(255, 255, 255);
    padding-top:5px;
}</string>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_button">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>510</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(135, 185, 225);
	color: rgb(170, 0, 0);
}
QPushButton::hover{
	background-color: rgb(5, 119, 161);
	color: rgb(255, 255, 255);
}
QPushButton::pressed{
	background-color: rgb(36, 83, 109);
    padding-left:5px;
	color: rgb(255, 255, 255);
    padding-top:5px;
}</string>
    </property>
    <property name="text">
     <string>EXÄ°T</string>
    </property>
   </widget>
   <widget class="QPushButton" name="true_button">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>430</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(135, 185, 225);
	color: rgb(170, 0, 0);
}
QPushButton::hover{
	background-color: rgb(5, 119, 161);
	color: rgb(255, 255, 255);
}
QPushButton::pressed{
	background-color: rgb(36, 83, 109);
    padding-left:5px;
	color: rgb(255, 255, 255);
    padding-top:5px;
}</string>
    </property>
    <property name="text">
     <string>TRUE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="false_button">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>430</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgb(135, 185, 225);
	color: rgb(170, 0, 0);
}
QPushButton::hover{
	background-color: rgb(5, 119, 161);
	color: rgb(255, 255, 255);
}
QPushButton::pressed{
	background-color: rgb(36, 83, 109);
    padding-left:5px;
	color: rgb(255, 255, 255);
    padding-top:5px;
}</string>
    </property>
    <property name="text">
     <string>FALSE</string>
    </property>
   </widget>
   <widget class="QLabel" name="true_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:solid;
border-width:3px;
border-radius:25px;
border-color: rgb(197, 103, 40);
color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="try_label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:solid;
border-width:3px;
border-radius:25px;
border-color: rgb(197, 103, 40);
color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>31</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>35</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(203, 102, 0);</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QLabel" name="level_label">
    <property name="geometry">
     <rect>
      <x>644</x>
      <y>425</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(135, 185, 225);
border-radius:10px;
color: rgb(170, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
